[2023-11-17T01:14:00.193+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: outcomes_dag.step1 manual__2023-11-17T01:13:58.996010+00:00 [queued]>
[2023-11-17T01:14:00.203+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: outcomes_dag.step1 manual__2023-11-17T01:13:58.996010+00:00 [queued]>
[2023-11-17T01:14:00.203+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 1
[2023-11-17T01:14:00.215+0000] {taskinstance.py:1382} INFO - Executing <Task(BashOperator): step1> on 2023-11-17 01:13:58.996010+00:00
[2023-11-17T01:14:00.220+0000] {standard_task_runner.py:57} INFO - Started process 81 to run task
[2023-11-17T01:14:00.225+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'outcomes_dag', 'step1', 'manual__2023-11-17T01:13:58.996010+00:00', '--job-id', '47', '--raw', '--subdir', 'DAGS_FOLDER/outcomes_dag.py', '--cfg-path', '/tmp/tmp4lbcg9ne']
[2023-11-17T01:14:00.226+0000] {standard_task_runner.py:85} INFO - Job 47: Subtask step1
[2023-11-17T01:14:00.304+0000] {task_command.py:416} INFO - Running <TaskInstance: outcomes_dag.step1 manual__2023-11-17T01:13:58.996010+00:00 [running]> on host c45f18363108
[2023-11-17T01:14:00.402+0000] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='outcomes_dag' AIRFLOW_CTX_TASK_ID='step1' AIRFLOW_CTX_EXECUTION_DATE='2023-11-17T01:13:58.996010+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2023-11-17T01:13:58.996010+00:00'
[2023-11-17T01:14:00.403+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2023-11-17T01:14:00.404+0000] {subprocess.py:75} INFO - Running command: ['/bin/bash', '-c', 'curl --create-dirs -o {CSV_TARGET_FILE} {SOURCE_URL}']
[2023-11-17T01:14:00.412+0000] {subprocess.py:86} INFO - Output:
[2023-11-17T01:14:00.423+0000] {subprocess.py:93} INFO -   % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
[2023-11-17T01:14:00.424+0000] {subprocess.py:93} INFO -                                  Dload  Upload   Total   Spent    Left  Speed
[2023-11-17T01:14:00.431+0000] {subprocess.py:93} INFO -   0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0curl: (6) Could not resolve host: SOURCE_URL
[2023-11-17T01:14:00.432+0000] {subprocess.py:97} INFO - Command exited with return code 6
[2023-11-17T01:14:00.440+0000] {taskinstance.py:1937} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/operators/bash.py", line 210, in execute
    raise AirflowException(
airflow.exceptions.AirflowException: Bash command failed. The command returned a non-zero exit code 6.
[2023-11-17T01:14:00.442+0000] {taskinstance.py:1400} INFO - Marking task as FAILED. dag_id=outcomes_dag, task_id=step1, execution_date=20231117T011358, start_date=20231117T011400, end_date=20231117T011400
[2023-11-17T01:14:00.449+0000] {standard_task_runner.py:104} ERROR - Failed to execute job 47 for task step1 (Bash command failed. The command returned a non-zero exit code 6.; 81)
[2023-11-17T01:14:00.480+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 1
[2023-11-17T01:14:00.504+0000] {taskinstance.py:2778} INFO - 0 downstream tasks scheduled from follow-on schedule check
